libraries/chain/webassembly/runtimes/eos-vm-oc.cpp

```diff
@@ -51,7 +62,11 @@ class eosvmoc_instantiated_module : public wasm_instantiated_module_interface {
 };
 
 eosvmoc_runtime::eosvmoc_runtime(const std::filesystem::path data_dir, const eosvmoc::config& eosvmoc_config, const chainbase::database& db)
-   : cc(data_dir, eosvmoc_config, db), exec(cc), mem(wasm_constraints::maximum_linear_memory/wasm_constraints::wasm_page_size) {
+   : cc(data_dir, eosvmoc_config, db) {
+   for (auto i = 0; i < 4; ++i) {
+      exec.emplace_back(std::make_unique<eosvmoc::executor>(cc));
```

Comment:
Ideally every executor in a thread shares the same resources. Not really sure best approach -- an executor could expose a `clone()` call to return a new executor sharing the same resources, or maybe there needs to be more refactoring such as a class that holds executor's resources that is just passed to a non-member `execute()` call (having trouble coming up with names for these things off hand).

I don't think we need to do this executor improvement immediately though

Reply:
Created an improvement issue for this https://github.com/AntelopeIO/spring/issues/1394